packages/office-ui-fabric-react/src/components/CommandBar/CommandBar.scss

```diff
@@ -193,6 +193,11 @@ $SearchBox-sidePadding: 8px; // padding in input on left and right sides without
   padding: 0 7px;
 }
 
+.divider {
+  background-color: $ms-color-neutralLight;
+  min-width: 1px;
```

Comment:
Would it make more sense to do this with a `border`, rather than a `background-color`?

Reply:
This is the same wait that ContextMenu renders its divider. I can change this or both dividers though if `border` is better.

Reply:
This is the way that ContextMenu renders its dividers. If a border is better, I can change the css in both locations.